benchmarks/src/main/java/io/grpc/benchmarks/qps/Utils.java

```diff
@@ -129,10 +131,27 @@ static SimpleRequest newRequest(ClientConfiguration config) {
   static Channel newClientChannel(ClientConfiguration config) throws IOException {
     if (config.transport == ClientConfiguration.Transport.OK_HTTP) {
       InetSocketAddress addr = (InetSocketAddress) config.address;
-      return OkHttpChannelBuilder
+      OkHttpChannelBuilder builder = OkHttpChannelBuilder
           .forAddress(addr.getHostName(), addr.getPort())
-          .executor(config.directExecutor ? MoreExecutors.newDirectExecutorService() : null)
-          .build();
+          .executor(config.directExecutor ? MoreExecutors.newDirectExecutorService() : null);
+      if (config.tls) {
+        SSLSocketFactory factory;
+        if (config.testca) {
+          builder.overrideHostForAuthority(TestUtils.TEST_SERVER_HOST);
```

Comment:
I believe Zhouhan is working on running this benchmark on Android, and she'll need this class.